.github/workflows/on-workflow-call-test.yml

```
@@ -9,11 +9,64 @@
+              exit 1
+            fi
+            echo "Attempt $attempt: Neon proxy not ready, waiting..."
+            sleep 2
+            attempt=$((attempt + 1))
+          done
+          echo "Successfully connected through Neon proxy!"
+
       - uses: actions/checkout@v4
       - uses: ./.github/actions/pnpm-install
         with:
           node-version: ${{ matrix.node-version }}
```

actionlint (1.7.7)

input "node-version" is not defined in action "pnpm install" defined at "./.github/actions/pnpm-install". available inputs are "node_version"

(action)